systematic-and-applied-microbiology.csl

```diff
@@ -112,14 +112,14 @@
     </choose>
   </macro>
   <citation collapse="citation-number">
-    <sort>
-      <key variable="citation-number" sort="ascending"/>
-    </sort>
     <layout delimiter="," prefix="[" suffix="]">
```

Comment:
I don't think that's right. With this deleted, you could get citations out of order in a single citation in the text, things like, [14, 3, 7, 2]. That can't be right. You can leave this in and still get alphabetical sorting in the bibliography.